utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -92,3 +92,70 @@ const val CACHE_LATEX_RENDERING = "cache_latex_rendering"
 
 /** Default value for whether to cache LaTeX rendering using Glide. */
 const val CACHE_LATEX_RENDERING_DEFAULT_VALUE = true
+
+/** Qualifier for the platform parameter that controls whether to record performance metrics. */
+@Qualifier
+annotation class PerformanceMetricsCollection
```

Comment:
Should this be 'EnablePerformanceMetricCollection' to more clearly indicate this as a feature flag? Ditto for its related fields/providers in this PR.

Reply:
done.